main.go

```diff
@@ -595,6 +598,13 @@ func main() {
 
 	authServer := grafana.NewAuthServer(grafanaClient, awsInstanceChecker)
 
+	// TODO:
+	dbaasControllerConn, err := grpc.Dial("127.0.0.1:20201", grpc.WithInsecure())
```

Comment:
maybe it's better to extract it to separate function like we did for qanClient

Reply:
Yes, that why TODO here.